crytic-config.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Did we want to add this?

Reply:
It's for the Crytic security checks. Will be updating it in a follow up PR